SwiftLSPClient/Types/Features/Reference.swift

```diff
@@ -21,7 +21,8 @@ public struct ReferenceParams: Codable {
     public let position: Position
     public let context: ReferenceContext
 
-    public init(textDocument: TextDocumentIdentifier, position: Position, context: ReferenceContext) {
+    public init(textDocument: TextDocumentIdentifier, position: Position,
+                context: ReferenceContext = ReferenceContext()) {
```

Comment:
What you've got here will work just fine. However, if instead you modify the following init method:

    public init(textDocument: TextDocumentIdentifier, position: Position, includeDeclaration: Bool)

to:

    public init(textDocument: TextDocumentIdentifier, position: Position, includeDeclaration: Bool = false)

You get the same API without affecting how `ReferenceContext` works. What do you think?

Reply:
D'oh, misunderstood you there. Committed a new commit.